OpenEdXMobile/src/main/java/org/edx/mobile/repositorie/InAppPurchasesRepository.kt

```diff
@@ -10,21 +9,11 @@ import org.edx.mobile.model.iap.ExecuteOrderResponse
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
-import roboguice.RoboGuice
+import javax.inject.Inject
+import javax.inject.Singleton
 
-class InAppPurchasesRepository {
-
-    private var iapAPI: InAppPurchasesAPI =
-        RoboGuice.getInjector(MainApplication.application)
-            .getInstance(InAppPurchasesAPI::class.java)
-
-    companion object {
-        private var instance: InAppPurchasesRepository? = null
-        fun getInstance(): InAppPurchasesRepository =
-            instance ?: synchronized(this) {
-                instance ?: InAppPurchasesRepository().also { instance = it }
-            }
-    }
+@Singleton
+class InAppPurchasesRepository @Inject constructor(private var iapAPI: InAppPurchasesAPI) {
```

Comment:
No need to inject a constructor here + making it singleton 
we can follow the same approach as using in CourseDateRepository